website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,241 @@ Learn more about Karpenter and how to get started below.
 * [Karpenter EKS Best Practices](https://aws.github.io/aws-eks-best-practices/karpenter/) guide
 * [EC2 Spot Workshop for Karpenter](https://ec2spotworkshops.com/karpenter.html)
 * [EKS Karpenter Workshop](https://www.eksworkshop.com/beginner/085_scaling_karpenter/set_up_the_provisioner/)
+
+This guide shows how to get started with Karpenter by installing and configuring it on a Kubernetes cluster.
+If that cluster is currently using Cluster Autoscaler, the guide also gives you the option of migrating away from Cluster Autoscaler.
+
+This guide assumes that you are starting with an existing Kubernetes cluster and that the following are true:
+
+* You will use existing VPC and subnets
+* You will use existing security groups
+* Your workloads have pod disruption budgets that adhere to [EKS best practices](https://aws.github.io/aws-eks-best-practices/karpenter/)
+* Your cluster has an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for service accounts
+* Your nodes may be part of one or more node groups
+
+This guide will also assume you have the `aws` and `kubectl` CLI tools installed.
+You can also perform many of these steps in the console, but we will use the command line for simplicity.
+
+## Get a cluster
+To use Karpenter, you must be running a Kubernetes cluster on a supported cloud provider.
+Currently, only AWS is supported.
+There are several choices of Kubernetes distributions and deployment tools that you can choose from.
+Here are examples:
+
+* [Creating an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html): Use `eksctl` or `aws` CLI tools or the AWS Management console to deploy an EKS cluster.
+* [Amazon EKS Blueprints for Terraform](https://aws-ia.github.io/terraform-aws-eks-blueprints): Follow a basic [Getting Started](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/getting-started/) guide and also add modules and add-ons. This includes a [Karpenter](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/add-ons/karpenter/) add-on that lets you bypass the instructions in this guide for setting up Karpenter.
+* [kOps](https://kops.sigs.k8s.io/operations/karpenter/): These instructions describe how to create a kOps Kubernetes cluster in AWS that includes Karpenter.
+
+If you have not already configured Karpenter when you set up your Kubernetes cluster in AWS, follow the instructions below to add Karpenter to your existing cluster.
+
+## Add Karpenter to an existing cluster
+
+To add Karpenter to your cluster, you need to cread IAM roles, add tags to subnets and security groups, and update the aws-auth ConfigMap.
+
+### Create IAM roles
+
+To get started, create two new IAM roles for nodes provisioned with Karpenter and the Karpenter controller.
```

Comment:
this can be simplified further to only requiring an IRSA role. Since Karpenter needs something to run on, then there already exists some form of compute that is connected to the control plane. tl;dr - there has to be at least one node with an IAM role and instance profile and the necessary permissions before Karpenter can even be provisioned. Users can simply re-use this role/instance profile. Just FYI

Reply:
I'm not sure exactly how to say that. Can you suggest specific wording?

Reply:
Here is what that could look like (changes based on your PR here) https://github.com/aws/karpenter/commit/b4ec84d65af52bc8c35cbdd871696577ad63e1ea

This cuts out quite a bit of overhead for users since there is already compute required before provisioning Karpenter which that means there is an existing IAM instance profile that is already in the `aws-auth` configmap (unless Karpenter is running on Fargate, in which case an IAM instance profile would need to be created). 

I will leave it up to you all to decided if this is worthwhile or not - just a suggestion to make this easier and with fewer resources/steps

Reply:
Thanks @bryantbiggs . I've incorporated the changes you suggested.